source/git-promise.js

```diff
@@ -374,14 +385,14 @@ git.commit = function(repoPath, amend, message, files) {
       }
     }
 
-    if (toAdd.length > 0) {
-      addPromise = git(['update-index', '--add', '--stdin'], repoPath, null, null, toAdd.join('\n'));
-    }
-    if (toRemove.length > 0) {
-      removePromise = git(['update-index', '--remove', '--stdin'], repoPath, null, null, toRemove.join('\n'));
-    }
+    var commitPromiseChain = new Promise(function(resolve) { resolve() }) // start with dummy
```

Comment:
You can use `Promise.resolve()` instead of `new Promise(function(resolve) { resolve() })`.